substrate/frame/support/procedural/src/benchmark.rs

```diff
@@ -808,6 +808,12 @@ pub fn benchmarks(
 						let elapsed_extrinsic = recording.elapsed_extrinsic().expect("elapsed time should be recorded");
 						let diff_pov = recording.diff_pov().unwrap_or_default();
 
+						// Time the storage root recalculation.
+						let start_storage_root = #krate::current_time();
+						#krate::__private::storage_root(#krate::__private::StateVersion::V1);
+						let finish_storage_root = #krate::current_time();
+						let elapsed_storage_root = finish_storage_root - start_storage_root;
+
```

Comment:
same: commit_db was caching storage root result. so first we need to measure storage_root itself.